community/contributing/pr_workflow.rst

```diff
@@ -54,11 +54,11 @@ The branches on the Git repository are organized as follows:
    occurs. As a development branch, it can be unstable
    and is not meant for use in production. This is where PRs should be done
    in priority.
--  The stable branches are named after their version, e.g. ``3.0`` and ``2.1``.
+-  The stable branches are named after their version, E.G. ``3.0`` and ``2.1``.
```

Comment:
I've never seen "i.e." and "e.g." spelled with capital letters.

https://simple.wikipedia.org/wiki/Exempli_gratia
https://simple.wikipedia.org/wiki/List_of_Latin_phrases_(I)#id_est